src/jupyter_contrib_nbextensions/nbextensions/codefolding/codefolding_editor.yaml

```diff
@@ -0,0 +1,12 @@
+Type: Jupyter Notebook Extension
+Name: Codefolding in Editor
+Description: |
+  Enables the CodeMirror feature to allow codefolding in the Jupyter file
+  editor view.
+  Note that this also uses the codefolding hotkey from the codefolding
+  nbextension for the notebook view.
```

Comment:
Yes,  although it'll be stored in edit config section rather than notebook, so will be independent of the notebook view one. Not a problem though, really.